roster2/tool/pom.xml

```diff
@@ -161,7 +161,7 @@
     <build>
         <plugins>
             <!-- compress CSS and JS -->
-            <plugin>
+            <!--plugin>
```

Comment:
Are we losing the minify feature for the Javascript and CSS files? Well..they are not very big but is a cool feature.

Reply:
Minify broke with the current closure compiler. I'm looking at that in library at the moment. I could add that back in, now, actually. I will do.